spec/system/candidate_interface/entering_details/candidate_entering_volunteering_experience_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'rails_helper'
+
+RSpec.feature 'Entering volunteering experience' do
+  include CandidateHelper
+
+  scenario 'Candidate adds volunteering experience' do
+    FeatureFlag.activate('restructured_work_history')
```

Comment:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/4213/files#r592394731